Dockerfile

```diff
@@ -1,44 +1,8 @@
-FROM golang:1.7-alpine
+FROM alpine:3.5
 
-COPY ./ /go/src/github.com/cloudposse/github-authorized-keys
+COPY ./releases/github-authorized-keys_linux_amd64 /usr/bin/github-authorized-keys
```

Comment:
Why not do the copy here https://github.com/cloudposse/github-authorized-keys/pull/24/files#diff-3254677a7917c6c01f55212f86c57fbfL77

Having a copy statement here will invalidate all subsequent cache layers